src/components/editMeasure/testCases/components/statusHandler/StatusHandlerMessage.tsx

```diff
@@ -86,18 +86,25 @@ export function createImportMessage(
   successfulImportsWithWarnings: TestCaseImportOutcome[],
   testDataId: string
 ) {
+  let failedMessage = `(${successfulImports}) test case(s) were imported. The following (
+              ${failedImports.length}) test case(s) could not be imported. Please
+              ensure that your formatting is correct and try again.`;
+  if (failedImports?.length > 0) {
+    const lockedTestCases = failedImports.filter((fi) =>
+      fi.message?.includes("The test case is locked by another user")
+    );
```

Comment:
`Array.some` is better if you just case unless you need `lockedTestCases` somewhere down in this function
```suggestion
    const lockedTestCases = failedImports.some((fi) =>
      fi.message?.includes("The test case is locked by another user")
    );
```

Reply:
@adongare this is changed to use some. Thanks so much Ashok!